### ISSUE TITLE
Player menus disappear before an item can be selected

### ISSUE THREAD

**Issue by ParaSait** [FreeTubeApp/FreeTube#1189]
**Behavior of the program**
When hovering the mouse over a widget that pops out a menu, then clicking an item in one of the pop-out menus in the video player controls (e.g. the quality menu), the menu disappears and the video gets paused or resumed. The issue can be prevented by clicking the widget first.

**Expected behavior**
Hovering over a pop-out menu widget and clicking a menu item selects the item and then disappears; the video doesn't pause or resume.

**To Reproduce**
1. Go to any video, and let it play.
2. Hover the mouse over the playback speed, closed caption or quality widget in the controls to cause the menu to pop out; do not click it.
3. Try to select any item in the menu that pops out (e.g., 720p in the quality menu) by clicking it.
4. As you click, the menu loses focus and disappears, the item did not get selected and the player settings are not modified, and the video gets paused.

**Environment Information (please complete the following information):**
 - OS: Arch Linux, under both xfce and qtile
 - FreeTube version: freetube-bin 0.12.0-3
 - Installation Method: AUR
 - Primary API Used: Local API

**Comment by sj-on** [comment]
Closed captions do not appear on video as well. I guess it is a result of your mentioned bug.

**Comment by GilgusMaximus** [comment]
This seems to be a Linux related issue.
On Windows everything is working fine.

@sj-on Could you try the newest nightly build. The caption system got a rework.

**Comment by sj-on** [comment]
@GilgusMaximus Sure. Will try.

**Comment by sj-on** [comment]
@GilgusMaximus The nightly builds aren't any better😅. I tried the latest one and not only did it not show captions, but also showed an API error, repeatedly (everytime I tried to play a video). My machine has Windows 10 running on it.

**Comment by GilgusMaximus** [comment]
Yeah, sorry for that. @Svallinn made a fix for it and I merged it 4 hours after your comment. So the local API error should be fixed at least
